lua/lvim/plugins.lua

```diff
@@ -265,11 +265,15 @@ local core_plugins = {
   -- Terminal
   {
     "akinsho/toggleterm.nvim",
-    event = "VeryLazy",
     branch = "main",
+    init = function()
+      require("lvim.core.terminal").init()
+    end,
     config = function()
       require("lvim.core.terminal").setup()
     end,
+    keys = lvim.builtin.terminal.keys,
```

Comment:
where is `lvim.builtin.terminal.keys` set?

Reply:
My bad, fixed